src/main/java/spring/intro/controllers/UserController.java

```diff
@@ -0,0 +1,49 @@
+package spring.intro.controllers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import spring.intro.dto.UserResponseDto;
+import spring.intro.model.User;
+import spring.intro.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping
+    public List<UserResponseDto> getAll() {
+        return userService.getListUsers().stream()
+                .map(this::getDto)
+                .collect(Collectors.toList());
+    }
+    
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+        return getDto(userService.getById(userId));
```

Comment:
nice try, bur checklist said: "don’t change service and dao layer"

Reply:
@gradiviro this is ok. We can create new methods, but we shouldn't return `dto` on the service layer